tests/e2e/Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+import groovy.json.JsonOutput
+
+/** Send a notice to #fxtest-alerts on irc.mozilla.org with the build result */
+def ircNotification() {
+  def nick = "fxtest${BUILD_NUMBER}"
+  def channel = '#fx-test-alerts'
+  def result = currentBuild.result ?: 'SUCCESS'
+  def message = "Project ${JOB_NAME} build #${BUILD_NUMBER}: ${result}: ${BUILD_URL}"
+  node {
+    sh """
+        (
+        echo NICK ${nick}
+        echo USER ${nick} 8 * : ${nick}
+        sleep 5
+        echo "JOIN ${channel}"
+        echo "NOTICE ${channel} :${message}"
+        echo QUIT
+        ) | openssl s_client -connect irc.mozilla.org:6697
+    """
+  }
+}
+
+pipeline {
+  agent any
+  options { timestamps() }
+  stages {
+    stage('Lint') {
+      steps {
+        sh "tox -c tests/e2e/tox.ini -e flake8"
+      }
+    }
+    stage('Test') {
+      steps {
+        timeout(time: 5, unit: 'MINUTES') {
+          wrap([$class: 'AnsiColorBuildWrapper']) {
```

Comment:
You can do the same with `timeout` if you're okay with it being for the pipeline as a whole:

options {
    ansiColor()
    timestamps()
    timeout time:5, units:'MINUTES'
}